### ISSUE TITLE
gam print browsers error

### ISSUE THREAD

**Issue by mpreilly924** [taers232c/GAMADV-XTD3#135]
The print browsers command is failing on versions 5.35.06 but working on version 5.31.10 

Error

ERROR:  JSON: {'error': {'code': 400, 'message': 'Invalid Customer ID: customers/my_customer', 'errors': [{'message': 'Invalid Customer ID: customers/my_customer', 'domain': 'global', 'reason': 'badRequest'}], 'status': 'INVALID_ARGUMENT'}}+

ERROR: 400: invalid - Invalid Customer ID: customers/my_customer

**Comment by taers232c** [comment]
Do:
```
$ gam info domain
Customer ID: C12at1m69
Primary Domain: domain.com
...
$ gam config customer_id C12at1m69 domain domain.com save
```

**Comment by taers232c** [comment]
I found a bug, will be fixed soon.

Ross

**Comment by taers232c** [comment]
Fixed 5.35.07

**Comment by mpreilly924** [comment]
Thank you for the quick response!!
